src/content/overview/observability/configuration/multi-tenancy/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Multi-tenancy
+description: Learn about multi-tenancy concepts and implementation in the Giant Swarm observability platform.
+weight: 10
+menu:
+  principal:
+    parent: overview-observability-configuration
+    identifier: overview-observability-configuration-multi-tenancy
+last_review_date: 2025-06-30
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is multi-tenancy in observability?
+  - How do I implement multi-tenancy?
+  - How do I create Grafana organizations?
+  - What are the multi-tenancy concepts?
+---
+
+Multi-tenancy in the Giant Swarm observability platform enables secure data isolation and access controls across teams, environments, and projects. This section covers both the conceptual foundation and practical implementation of multi-tenancy.
+
+## Understanding multi-tenancy
+
+Learn the core concepts behind multi-tenancy in the observability platform:
+
+- **[Multi-tenancy concepts]({{< relref "/overview/observability/configuration/multi-tenancy/concepts" >}})**: Understand tenants, organizations, and how they work together to provide data isolation and access control
+
+## Implementing multi-tenancy
+
+Put multi-tenancy into practice:
+
+- **Creating Grafana organizations**: Step-by-step guide to create and configure Grafana organizations for your teams
+
+## Getting started
```

Comment:
Why have previous sections if `Getting started` had those? Seems repetitive to me

Reply:
I can remove this yes